src/components/Nav/Nav.js

```diff
@@ -24,18 +30,86 @@ class Nav extends Component {
               />
             </Link>
           </div>
+          <nav className="p-nav__nav_links--mobile">
+            <span
+              className="p-nav__dropdown_mobile"
+              onClick={() => this.setState({ showing: !showing })}
+            >
+              Menu
+              {showing ? (
+                <div className="p-nav__dropdown_mobile--content">
+                  <div className="p-nav__dropdown--account">
+                    <NavLink
+                      exact
+                      to="/apply"
+                      className="p-nav__dropdown_link--apply"
+                    >
+                      Apply
+                    </NavLink>
+                    <NavLink exact to="/login" className="p-nav__dropdown_link">
+                      Login
+                    </NavLink>
+                  </div>
+                  <hr />
+                  <NavLink exact to="/faq" className="p-nav__dropdown_link">
+                    FAQ
+                  </NavLink>
+                  <NavLink exact to="/hackers" className="p-nav__dropdown_link">
+                    Hackers
+                  </NavLink>
+                  <NavLink
+                    exact
+                    to="/sponsors"
+                    className="p-nav__dropdown_link"
+                  >
+                    Sponsors
+                  </NavLink>
+                  <NavLink exact to="/team" className="p-nav__dropdown_link">
+                    Team
+                  </NavLink>
+                  <NavLink exact to="/about" className="p-nav__dropdown_link">
+                    About
+                  </NavLink>
+                  <NavLink exact to="/slack" className="p-nav__dropdown_link">
+                    Join our Slack
+                  </NavLink>
+                </div>
+              ) : null}
+            </span>
+          </nav>
+
           <nav className="p-nav__nav_links">
-            <NavLink exact to="/hackers" className="p-nav__nav_link">
-              Hackers
+            <div className="p-nav__dropdown">
+              <NavLink exact to="/about" className="p-nav__nav_link">
+                About ▾
+              </NavLink>
+              <div className="p-nav__dropdown_content">
+                <NavLink exact to="/hackers" className="p-nav__dropdown_link">
+                  Hackers
+                </NavLink>
+                <NavLink exact to="/sponsors" className="p-nav__dropdown_link">
+                  Sponsors
+                </NavLink>
+                <NavLink exact to="/team" className="p-nav__dropdown_link">
+                  Team
+                </NavLink>
+              </div>
+            </div>
+            <NavLink exact to="/faq" className="p-nav__nav_link">
+              FAQ
             </NavLink>
-            <NavLink exact to="/sponsors" className="p-nav__nav_link">
-              Sponsors
+            <NavLink exact to="/slack" className="p-nav__nav_link">
```

Comment:
Can we remove the slack nav item for now. Might move this to the dashboard. Unsure